source/SpinalHDL/Libraries/stream.rst

```diff
@@ -534,6 +534,27 @@ This util take its input stream and routes it to ``outputCount`` stream in a seq
      outputCount = 3
    )
 
+StreamTransactionExtender
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This util helps to extend one input transaction with specified ``count``+1 times of output transactions with the same payload.
```

Comment:
This utility will take one input transaction and generate one output transaction, it provides the facility to repeat the payload value ``count``+1 times repeated into the same output transaction.

The summary doesn't make it immediately understood what is does.
Use of the term 'helps' makes it sound like it is a partial solution to a problem and is not the free standing solution that it is.

Can we provide a wavedrom of an example of the signalling ?